wrangler-core/src/main/java/io/cdap/wrangler/utils/ColumnConverter.java

```diff
@@ -203,8 +205,10 @@ private static Object convertType(String col, String toType, Object object) thro
           return Bytes.toString((byte[]) object);
         } else if (object instanceof Boolean) {
           return Boolean.toString((Boolean) object);
+        } else if (object instanceof LocalDateTime) {
+          return ((LocalDateTime) object).format(DateTimeFormatter.ISO_DATE_TIME);
         } else {
-          return object;
+          break;
```

Comment:
I wonder if string should just return object.toString()? I think we would be able to get rid of a bunch of these instanceof checks, unless Double.toString() behaves differently?

Also, I don't see a null check anywhere, is null checked outside of this method? Or does it throw an exception?

Reply:
Null check is done before calling this. Yes, I think a toString() should be fine except for byte[].  Double and other wrappers toString() calls the static toString(), so it should be fine. For Datetime toString() gives a ISO-8601 format. So will allow all types to be converted to string.

Reply:
Done.